kong/clustering/data_plane.lua

```diff
@@ -74,11 +74,20 @@ function _M.new(clustering)
 end
 
 
+local function set_control_plane_connected(reachable)
+  local ok, err = ngx.shared.kong:safe_set("control_plane_connected", reachable)
+  if not ok then
+    ngx_log(ngx_ERR, _log_prefix, "failed to set controlplane_reachable key in shm to " .. reachable .. " :", err)
```

Comment:
Please do not use string concatenation in `ngx.log()` ([docs](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#code-style))

```suggestion
    ngx_log(ngx_ERR, _log_prefix, "failed to set controlplane_reachable key in shm to ", reachable, " :", err)
```